### ISSUE TITLE
Projectiles fired from the same event share the same lifetime as the first projectile fired

### ISSUE THREAD

**Issue by Logzombie** [CapsAdmin/pac3#858]
A little more context: Once a projectile is fired via an event (E.G Button event), the lifetime of the first projectile fired dictates the current lifetime for all existing projectiles from the same triggered event. Once the first runs out, it despawns all projectiles fired afterwards, regardless of if they're mid-flight or not.

See video for example: https://dl.dropboxusercontent.com/s/shzq488ji186r7c/Garrysmod%202020.01.30%20-%2009.19.10.02.compressed.mp4

**Comment by Yagira** [comment]
You sure about this?
https://streamable.com/vt80s

**Comment by Logzombie** [comment]
I'll have to post my exact setup when I can, perhaps it's an issue regarding the inclusion of TimerX. I'll do some trial and error and update the report if need be.

**Comment by Logzombie** [comment]
The setup: 
![image](https://user-images.githubusercontent.com/50374409/73850447-993a2000-4823-11ea-86a4-8a7a6934384c.png)

**Comment by Yagira** [comment]
Nope.
https://streamable.com/b8tct

**Comment by Logzombie** [comment]
...

Ok give me two seconds. Now I'm baffled.

**Comment by Logzombie** [comment]
Ah, think I've found the problem, it's related to the projectile outfit itself, not the projectile part or event.

The projectile outfit setup: 
![image](https://user-images.githubusercontent.com/50374409/73859895-add1e480-4832-11ea-996a-ff84636e1fc9.png)

**Comment by Logzombie** [comment]
Yep! That's the issue it seems. Gonna update the issue in a second. This became such a common occurrence in my PACs that I honestly thought it was happening on basically every event.

https://dl.dropboxusercontent.com/s/jkz07php97l49ff/Garrysmod%202020.02.05%20-%2016.17.42.02.compressed.mp4

**Comment by CapsAdmin** [comment]
Can you also upload an outfit that illustrates the problem?

**Comment by Yagira** [comment]
There you go. [projectile.txt](https://github.com/CapsAdmin/pac3/files/4164517/projectile.txt)
In short, if the group used by the projectile (projectile1-group) has an event that triggers another projectile (projectile2) AND the second projectile is set as physical, all projectiles disappear once the original timer (projectile1 life time) runs out.

**Comment by Logzombie** [comment]
Coming back to this, this seems to still be an apparent issue for projectiles included inside projectile outfits. I'm unsure if this affects projectiles server-side as opposed to just the PAC firing said projectile. This only occurs on physical projectiles, while non-physical projectiles seem to avoid this issue.
